build/scripts/previousversion/main.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
Ooh, this file needs an Apache header please.

Reply:
I didn't get this, what do you mean by apache header?

Reply:
We need the open source Apache licence header at the top of all code files.

https://www.apache.org/licenses/LICENSE-2.0#apply

For example, see https://github.com/googleforgames/agones/blob/main/examples/xonotic/main.go